crates/core/src/interpreter/backend/resolve_logs.rs

```diff
@@ -0,0 +1,63 @@
+use crate::common::{query_result::LogQueryRes, types::LogField};
+use std::error::Error;
+use alloy::{
+    providers::{Provider, RootProvider}, rpc::types::Filter, transports::http::{Client, Http}
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, thiserror::Error)]
+pub enum LogResolverErrors {
+    // #[error("Invalid address")]
+    // InvalidAddress,
+    #[error("Mismatch between Entity and EntityId")]
+    MismatchEntityAndEntityId,
```

Comment:
since now we have a specific error for this module we can add more context here, since the error message doesn't need be generic anymore. 

```suggestion
    #[error("Entity id {0} can't be used to identify Log")]
    MismatchEntityAndEntityId(String),
```

Reply:
Logs can't have a entity_id. I'm not even testing this at this file, I can remove this. Are you suggesting I need to check it?

Reply:
This comment is also relevant: https://github.com/iankressin/eql/pull/39#discussion_r1752627826

Reply:
I forgot logs doesn't have ids. In this case we need can use a simple message like "Mismatch filters for Logs".

In the future I want to remove all verification between entities and entity ids from the `backend`, this is should happen on the `frontend` only. Therefore we don't need to waste too much time doing analysis of which filter is malformatted right now.

This was a poor design decision from my part.